lua/clangd_extensions/memory_usage.lua

```diff
@@ -21,19 +21,21 @@ local function display(lines)
         col = col,
         border = require("clangd_extensions.config").options.memory_usage.border,
     })
-    vim.bo.shiftwidth = 2
+    vim.bo[buf].shiftwidth = 2
```

Comment:
can you change this one to `nvim_set_option_value` too? i don't know why i did it this way

Reply:
> can you change this one to `nvim_set_option_value` too? i don't know why i did it this way

@p00f Sure! On it rn. Should I modify line 25 too (`vim.wo.foldmethod = "indent"`)?

I can do it as shown below:

![2024-08-24-195301_hyprshot](https://github.com/user-attachments/assets/3bf777b2-54c0-4edd-8cf6-7f123e3fd8ff)